client/mixins/imageHelper.ts

```diff
@@ -264,82 +248,248 @@ const imageHelper = Vue.extend({
         }
       }
     },
+    getDesiredAspect(shape: CardShapes): number {
+      let aspectRatio;
+
+      switch (shape) {
+        case 'portrait-card':
+          aspectRatio = 2 / 3;
+          break;
+        case 'thumb-card':
+          aspectRatio = 16 / 9;
+          break;
+        case 'banner-card':
+          aspectRatio = 1000 / 185;
+          break;
+        case 'square-card':
+        default:
+          aspectRatio = 1;
+          break;
+      }
+
+      return aspectRatio;
+    },
     /**
-     * Returns the URL of an item's image:
-     * · When 'element' parameter is passed, size of the image will be determined by the element's width & height
-     * · When 'maxWidth' and 'maxHeight' parameters are passed, size of the image will be as requested
-     * · When no 'element' or 'maxWidth' or 'maxHeight' is provided, image will have the original size.
+     * Generates the image information for a BaseItemDto or a BasePersonDto according to set priorities.
+     *
+     * @param {(BaseItemDto | BaseItemPerson)} item - Item to get image information for
+     * @param {Object} [options] - Options object
```

Comment:
```suggestion
     * @param {object} options - Optional parameters for the function.
```

I think the original description was more suitable and it also fixes the lint problem :P.